ToolBox/ZerodhaDownloader/ZerodhaDataDownloaderProgram.cs

```diff
@@ -38,29 +38,29 @@ public class ZerodhaDataDownloaderProgram
         /// </summary>
         public static void ZerodhaDataDownloader(IList<string> tickers, string market, string resolution, string securityType, DateTime startDate, DateTime endDate)
         {
+
             if (resolution.IsNullOrEmpty() || tickers.IsNullOrEmpty())
             {
-                Console.WriteLine("ZerodhaDataDownloader ERROR: '--tickers=', --securityType, '--market' or '--resolution=' parameter is missing");
-                Console.WriteLine("--tickers=eg JSWSTEEL,TCS,INFY");
-                Console.WriteLine("--market=MCX/NSE/NFO/CDS/BSE");
-                Console.WriteLine("--security-type=Equity/Future/Option/Commodity");
-                Console.WriteLine("--resolution=Minute/Hour/Daily/Tick");
+                Log.Error("ZerodhaDataDownloader ERROR: '--tickers=', --securityType, '--market' or '--resolution=' parameter is missing");
+                Log.Error("--tickers=eg JSWSTEEL,TCS,INFY");
+                Log.Error("--market=MCX/NSE/NFO/CDS/BSE");
+                Log.Error("--security-type=Equity/Future/Option/Commodity");
+                Log.Error("--resolution=Minute/Hour/Daily/Tick");
                 Environment.Exit(1);
             }
             try
             {
                 var _kite = new Kite(_apiKey, _accessToken);
+                var _symbolMapper = new ZerodhaSymbolMapper(_kite);
```

Comment:
nit - We add `_` for private class members, these should just be `kite` and `symbolMapper`